src/cmds/qsub.c

```diff
@@ -2346,21 +2346,15 @@ process_opts(int argc, char **argv, int passet)
 	int ddash_index = -1;
 
 #ifdef WIN32
-#define GETOPT_ARGS_ORIG "a:A:c:C:e:fGhIj:J:k:l:m:M:N:o:p:q:r:R:S:u:v:VW:zP:"
+#define GETOPT_ARGS "a:A:c:C:e:fGhIj:J:k:l:m:M:N:o:p:q:r:R:S:u:v:VW:zP:"
 #else
 #if !defined(PBS_NO_POSIX_VIOLATION)
-#define GETOPT_ARGS_ORIG "a:A:c:C:e:fhIj:J:k:l:m:M:N:o:p:q:r:R:S:u:v:VW:XzP:"
+#define GETOPT_ARGS "a:A:c:C:e:fhIj:J:k:l:m:M:N:o:p:q:r:R:S:u:v:VW:XzP:"
 #else
-#define GETOPT_ARGS_ORIG "a:A:c:C:e:fhj:J:k:l:m:M:N:o:p:q:r:R:S:u:v:VW:zP:"
+#define GETOPT_ARGS "a:A:c:C:e:fhj:J:k:l:m:M:N:o:p:q:r:R:S:u:v:VW:zP:"
 #endif /* PBS_NO_POSIX_VIOLATION */
 #endif /* WIN32 */
 
-#ifdef PBS_GNU_GETOPTS
-#define GETOPT_ARGS "+"GETOPT_ARGS_ORIG
-#else
-#define GETOPT_ARGS GETOPT_ARGS_ORIG
-#endif /* PBS_GNU_GETOPTS */
```

Comment:
Did PBS_GNU_GETOPTS not get defined previously?  Is this basically dead code?

Reply:
PBS_GNU_GETOPTS never gets defined, so I have removed the dead code. PBS_GNU_GETOPTS used to get defined by the configure script by compiling a little .c program and checking the return code, but from looking at it, I don't think it tested the option correctly.